src/main/java/de/dhbw/Main.java

```diff
@@ -61,6 +62,9 @@ public static void runController(){
                     switch (setting.getType()) {
```

Comment:
what if we add another message type for the Control-Marker-Settings like
```suggestion
                    switch (setting.getType()) {
                           case CM_VELOCITY:
                                 UIMessage veloMsg = new UIMessage(...)
                                 // NO break
                          case VELOCITY:
                                midiAdapter.setVelocity(...)
                                break;
```

Reply:
That's not really atomic information, right? Do you think an additional field would bloat the messages unnecessarily?

Reply:
I think in terms clean-code another message-type would be ugly, but I think in terms of control-flow sending the UI-Message more often than necessary is even uglier, 
but since it does not happen very often anyways it does not really matter which way we choose

Reply:
Then let's go with your way for simplicity as you proposed above. I guess we can always change that later ^^

With that this PR should be ready to merge, right?